### DISCUSSION TITLE
How to format phone number with localized digits?

### DISCUSSION THREAD

**Discussion by Shayan-To** [intlify/vue-i18n#911] [Q&A]
I know I can use `n` to format numbers with localized digits (so that `1234567890` becomes `۱۲۳۴۵۶۷۸۹۰`). What I don't know is how I can format phone numbers (like `09123456789`) with localized digits (so it becomes `۰۹۱۲۳۴۵۶۷۸۹`). What is the correct way of doing that?

**Comment by kazupon** [comment]
Internally, `n` uses the `Iintl.NumberFormat` API internally, so what you can localize with `n` depends on it.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat

It is recommended to use the i18n library for phone numbers.

**Reply by Shayan-To** [reply]
> It is recommended to use the i18n library for phone numbers.

What function should I call? Could you please give me an example?
